src/commands/deploy/nuxt/deploy.ts

```diff
@@ -27,20 +27,31 @@ import {
     CreateFrontendV2Path,
 } from "../../../requests/createFrontendProject.js";
 import { DeployCodeFunctionResponse } from "../../../models/deployCodeResponse.js";
+import { DeployType } from "../command.js";
 
-export async function nuxtDeploy(options: GenezioDeployOptions) {
+export async function nuxtNitroDeploy(options: GenezioDeployOptions, deployType: DeployType) {
```

Comment:
The real signature you want is:
```suggestion
export async function nuxtNitroDeploy(options: GenezioDeployOptions, deployType: DeployType.Nitro | DeployType.Nuxt) {
```